internal/util/update/update_test.go

```diff
@@ -703,6 +704,93 @@ func TestCommand_Run_failInvalidPath(t *testing.T) {
 	}
 }
 
+func TestCommand_Run_badUpstreamLock(t *testing.T) {
+	testCases := map[string]struct {
+		dir              string
+		commit           string
+		repo             string
+		expectedErrorMsg string
+	}{
+		"bad commit": {
+			dir:              "/",
+			commit:           "fake",
+			repo:             testutil.Upstream,
+			expectedErrorMsg: "unknown revision or path"},
+		"bad dir": {
+			dir:              "/fake",
+			commit:           "",
+			repo:             testutil.Upstream,
+			expectedErrorMsg: "no such file or directory"},
+		"bad repo": {
+			dir:              "/",
+			commit:           "",
```

Comment:
Nit: Just leave them out instead of setting the nil value.

Reply:
Done.